Gopkg.toml

```diff
@@ -142,7 +142,7 @@ required = [
   name = "github.com/nats-io/nats-streaming-server"
   version = "0.11.0"
 
-# The latest release as of March 21, 2019.
+# The latest release as of March 27, 2019.
```

Comment:
do we really need the release date ?

Reply:
+1 now that they are versioned, seems like it's not necessary. When we use a sha makes sense to include the date?